src/__stories__/config.js

```diff
@@ -14,9 +14,11 @@ Events.onEvent(data => {
   console.log("Tracked event", data)
 })
 
-setOptions({
-  name: "Reaction",
-  url: "http://artsy.github.io/reaction",
-  showDownPanel: false,
-  sortStoriesByKind: true,
-})
+
+// TODO: Fix the below
```

Comment:
@orta this seems to be broken with the latest version of storybook- would you be able to look into it?